source/features/select-all-notifications.ts

```diff
@@ -0,0 +1,16 @@
+import elementReady from 'element-ready';
+import * as pageDetect from 'github-url-detection';
+
+import features from '.';
+
+async function init(): Promise<void> {
+	(await elementReady<HTMLElement>('.js-notifications-mark-all-prompt'))!.dataset.hotkey = 'a';
```

Comment:
Seems like the problem mentioned in https://github.com/sindresorhus/refined-github/issues/3876#issuecomment-757399774 (the "Select all" checkbox being removed) doesn't occur anymore, so the feature becomes even simpler.

Reply:
What if I don't have any notifications?

![image](https://user-images.githubusercontent.com/16872793/111478150-2c6fa580-8706-11eb-8095-53b3d964583b.png)

Reply:
> What if I don't have any notifications?

You mean what if new notifications appear when the page is open? I'd have to check but I'm pretty sure clicking the `x new notification(s)` link fully reloads the page.

Reply:
> Edit: or do you mean we should exclude an empty notification page?

Yes, is there a way to do that domReady safe?

Reply:
I think it's easier to just check the result of `elementReady`?